atorch/examples/ds_3d_parallel/llama2_entry.sh

```diff
@@ -0,0 +1,56 @@
+set -x
+
+DATASET_PATH=${DATASET_PATH:-/path/to/wikitext-2-raw-v1}
```

Comment:
Any script that can download dataset/model if users don't have it ?

Reply:
Done